browser/version_info.h

```diff
@@ -7,6 +7,11 @@
 
 #include <string>
 
+#define MESSAGE_NOTIFIER_TYPE message_center::NotifierType::SYSTEM_COMPONENT
```

Comment:
can you give me some info why is this used?

Reply:
not sure how this is related to rewards

Reply:
@samartnik I guess we can remove that as it was used in transitions before chromium version when core was on a higher version and android on stable lower.

Reply:
Removed